src/components/LayoutFooter/logo.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="36px" height="22px" viewBox="0 0 36 22" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
same here, do we remove it from other places?

Reply:
Deduplicated it too